src/plone/volto/default_homepage/demo.py

```diff
@@ -349,271 +349,538 @@
+                                "type": "li"
                             },
                             {
-                                "type": "li",
                                 "children": [
-                                    {"text": "The "},
                                     {
-                                        "type": "link",
-                                        "data": {"url": "https://plone.org/"},
-                                        "children": [{"text": "Plone community "}],
+                                        "text": ""
                                     },
-                                    {"text": "web site for developers."},
+                                    {
+                                        "children": [
+                                            {
+                                                "text": "Plone Training"
+                                            }
+                                        ],
+                                        "data": {
+                                            "url": "https://training.plone.org/"
+                                        },
+                                        "type": "link"
+                                    },
+                                    {
+                                        "text": ""
+                                    }
                                 ],
+                                "type": "li"
                             },
                             {
-                                "type": "li",
                                 "children": [
-                                    {"text": "The "},
                                     {
-                                        "type": "link",
-                                        "data": {"url": "https://www.python.org/"},
+                                        "text": ""
+                                    },
+                                    {
                                         "children": [
-                                            {"text": "Python programming language"}
+                                            {
+                                                "text": "Forum"
+                                            }
                                         ],
+                                        "data": {
+                                            "url": "https://community.plone.org/"
+                                        },
+                                        "type": "link"
                                     },
-                                    {"text": " web site."},
+                                    {
+                                        "text": ""
+                                    }
                                 ],
+                                "type": "li"
                             },
+                            {
+                                "children": [
```

Comment:
I'm not sure which list to provide here, maybe both: awesome Plone add-ons(backend) and also awesome Volto add-ons (https://github.com/collective/awesome-volto#awesome-volto). What do you think? If both lists, than maybe a short explanation would be helpful.

Reply:
Fixed in 2dc3c7a90e4574323259d9792b1ea0c81704c7e5